extensions/cdc-postgres/src/test/java/com/hazelcast/jet/cdc/postgres/MultiTableCacheIntegrationTest.java

```diff
@@ -287,7 +231,7 @@ private static class OrderEntryProcessor implements EntryProcessor<Integer, Orde
         @Override
         public Object process(Entry<Integer, OrdersOfCustomer> entry) {
             try {
-                boolean deletion = Operation.DELETE.equals(record.operation());
+                boolean deletion = DELETE.equals(record.operation());
```

Comment:
```suggestion
                boolean deletion = DELETE == record.operation();
```

... to keep it in sync with other enum usages in the class.